crates/ty_python_semantic/resources/mdtest/scopes/global.md

```diff
@@ -211,3 +211,16 @@ def f():
     global x
     x: str = "foo"  # TODO: error: [invalid-syntax] "annotated name 'x' can't be global"
 ```
+
+## Global declarations affect the inferred type of the binding
+
+Even if the `global` declaration isn't used in an assignment, we conservatively assume it could be:
+
+```py
+x = 1
+
+def f():
+    global x
+
+# TODO: reveal_type(x)  # revealed: Unknown | Literal["1"]
```

Comment:
Should I open an issue for this TODO?

Reply:
Deferring it for now and opening a followup sounds great to me!